tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -632,7 +634,7 @@ def test_config_interface_state(self, setup_config_mode, sample_intf):
         if out['rc'] != 0:
             pytest.fail()
 
-        wait(3)
+        wait(PORT_TOGGLE_DELAY)
```

Comment:
Probably use wait_until with an even large timeout?

Reply:
Good suggestion. I refactored the code. Thanks!